### ISSUE TITLE
keras fit_generator ignores verbose parameter for validation_data

### ISSUE THREAD

**Issue by WaeCo** [tensorflow/tensorflow#28738]
Setting `verbose=0` in `tf.keras.model.fit_generator()` has no effect if `validation_data` is provided.
The progress is still displayed for each validation step.
This is annoying if you have a callback that is responsible for printing progress.

**System information**
- running in docker container tensorflow/tensorflow:latest-py3-jupyter
- `tf.GIT_VERSION`: v1.13.1-0-g6612da8951
- `tf.VERSION`: 1.13.1 
- `tf.keras.__version__`: 2.2.4-tf

**Describe the current behavior**
Progress-bar is displayed regardless of `verbose=0`

**Describe the expected behavior**
Nothing should be printed

**Code to reproduce the issue**
```python
model = tf.keras.models.Sequential([
    tf.keras.layers.Dense(1, input_shape=(1,))
])
model.compile(loss="mae", optimizer="adam")

def generator():
    i=0
    while 1:
        yield (np.array([i]),[i])
        i+=1
valData = (np.arange(10), np.arange(10))

history = model.fit_generator(generator(), steps_per_epoch=5, verbose=0, validation_data=valData)
```
Output:
> 10/10 [==============================] - 0s 6ms/sample - loss: 11.3572

Omitting `validation_data`  stops the output from appearing

Link to an example notebook:
https://colab.research.google.com/drive/1SrTdFXD_SCxu-gvo-h2YjFiAkO1CDaLG

**Comment by gadagashwini** [comment]
@WaeCo This is duplicate of [#28648](https://github.com/tensorflow/tensorflow/issues/28648).

**Comment by WaeCo** [comment]
You are right, apparently I did not see this in my search
